_posts/en/newsletters/2021-03-24-newsletter.md

```diff
@@ -0,0 +1,299 @@
+      verification up to three times faster.  It also [makes][poelstra
+      slowdowns] anonymous membership proofs and related techniques both
+      harder to develop and much more CPU intensive.  There
+      [may][poelstra patent story] also be a patent issue.
+
+  As of this writing, it appears the mailing list discussion has
+  concluded without any obvious loss of community support for taproot.
+  As researchers and businesses continue improving the state of the art
+  in quantum computing, we expect to see future discussions about how to
+  best keep Bitcoin secure.
+
+- **Weekly taproot activation meetings:** ten weekly meetings to discuss
+  details related to activating [taproot][topic taproot] have been
+  scheduled for each Tuesday at 19:00 UTC in the
+  [##taproot-activation][] IRC channel, with the first
+  [meeting][activation meeting log] occurring yesterday (March 23rd).
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [C-Lightning 0.10.0-rc1][C-Lightning 0.10.0] FIXME:harding
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #20861][] BIP 350: Implement Bech32m and use it for v1+ segwit addresses FIXME:Xekyo
+
+- [Bitcoin Core #21141][] updates the `-walletnotify` configuration
+  setting that calls a user-specified command each time a transaction is
+  seen that affects a loaded wallet.  Two new placeholders are added to
+  the arguments that can be passed to the command, `%b` for the hash of
+  a block containing the transaction and `%h` for the height of the
+  block.  Both are set to placeholder values for unconfirmed
+  transactions.
+
+- [C-Lightning #4428][] deprecates the `fundchannel_complete` RPC's
+  acceptance of txids, requesting instead that a [PSBT][topic psbt] be
+  passed.  The PSBT can be checked to ensure it contains the funding
+  output, eliminating a [problem][c-lightning #4416] where a user who
+  passes incorrect data can lose the ability to recover their funds.
+
+- [C-Lightning #4421][] Experimental option option_shutdown_wrong_funding: help me, I screwed up! FIXME:dongcarl (suggestion: link to last week's newsletter about this topic)
+
+- [LND #5064][] allows setting the desired number of blocks it should
```

Comment:
Looks like this might get partially reverted: https://github.com/lightningnetwork/lnd/pull/5064#commitcomment-48478012. Not sure whether we should still cover it.

Reply:
I'll drop it from this newsletter and we can put it in next week's newsletter if it's still relevant.  Nice catch!